src/metrics/mod.rs

```diff
@@ -131,12 +176,9 @@ macro_rules! heatmap {
             name = $name,
             crate = metriken
         )]
-        pub static $ident: metriken::Heatmap = metriken::Heatmap::new(
-            0,
-            8,
+        pub static $ident: metriken::AtomicHistogram = metriken::AtomicHistogram::new(
```

Comment:
I think we should rename the macro as well from `heatmap!` else it is a bit confusing.

Reply:
Fixed.